readme.md

```diff
@@ -0,0 +1,62 @@
+# Load native code based on RDF descriptions
+
+This package implements a loader registry, which can be used to load code declared in RDF triples.
+
+As the name implies, you'll need actual loaders to do the job. One such package is 
```

Comment:
The name doesn't imply much to me (no big deal though, it's just that seeing `rdf-native-loader` doesn't make it obvious to me that I'll need other loaders to load other things ;) )

Reply:
You're right, this doesn't make sense.

Reply:
Oh, now I understand what this package is supposed to do / why we need it.

What do you think of renaming it? It exposes a class named `LoaderRegistry`, `rdf-loader-registry`?

Reply:
Is it about the missing `registry` in the name or also about the `native`?

We had already a discussion about the name and the `native` was added, as it sounded to much like something which loads triples/datasets without it.

Adding `registry` sounds good to me.

Reply:
I think developers can guess what a loader is, and usually what it can load is pretty clear. Here we have either a loader for rdf native in which case what rdf native means is not clear, or a loader that is rdf native in which case we don't know what it can load.

Another approach might be: how would you describe what this package does and why we need it in a couple of sentences?

Reply:
That's how I would describe it:

The package provides a registry for loaders, which return native JS objects (primitives, functions, plain objects, ...) for a given RDF based description. The common interface of the loaders in combination with the registry simplifies the recurring process of RDF to JS native object mapping in a reusable way.

Reply:
I see, thanks. This means I forgot the following option: rdf native loader is neither an rdf-native loader nor loader for rdf-native, rather they are native loaders for rdf. It proves how confusing this name was.

I suggest `rdf-loaders-registry`